### PR TITLE
Awesome books: with ES6 PR

### PR DESCRIPTION
**Pull request by belmeetmule**
### In this PR, I have accomplished the following tasks:

- [ ] refactor awesome-books app to use more classes
- [ ] add classes into modules 
- [ ] refactor methods of the classes to use arrow functions
- [ ] Added and used luxon date-time library

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
+// import modules from other sources
+import { DateTime as localDate } from './modules/luxon.js';
+import Book from './modules/Book.js';
+import UI from './modules/ui.js';
+
+const dateTime = document.querySelector('.date-time');
+// dateTime.innerHTML = new Date().toLocaleString();
```
Comment (vechicin) [inline index.js]: Please remove this commented line of code to keep a cleaner codebase.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+const addList = document.getElementById('add');
+const contact = document.getElementById('contact');
+const displaySection = document.getElementById('book-list');
+const inputSection = document.querySelector('.book-input');
+const contactSection = document.querySelector('.contact-section');
+
+// eslint-disable-next-line max-classes-per-file
+
+class Collection {
+  // eslint-disable-next-line no-unused-vars
```
Comment (vechicin) [inline index.js]: Please do not disable ESLint rules unless they are in the Luxon file or are related to the number of classes per file.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  contactSection.style.display = 'none';
+  e.preventDefault();
+});
+
+const deletebtn = document.querySelectorAll('.remove');
+
+deletebtn.forEach((btn) => {
+  btn.addEventListener('click', (e) => {
+    if (e.target.class === 'remove') {
+      // UI.deleteBook(e.target);
```
Comment (vechicin) [inline index.js]: Please remove this line of commented code to keep a clean codebase.

File: index.js [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+`;
+  e.preventDefault();
+
+  // for add button
+  const addbtn = document.querySelector('.btn');
+  addbtn.addEventListener('click', (e) => {
+    const title = document.querySelector('.title').value;
+    const author = document.querySelector('.author').value;
+    if (title === '' || author === '') {
+      // eslint-disable-next-line no-alert
```
Comment (vechicin) [inline index.js]: Please do not disable ESLint rules unless they are in the Luxon file or are related to the number of classes per file.

File: index.js [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+      alert('Please fill in all fields');
+    } else {
+      const newBook = new Book(title, author);
+      myCollection.add(newBook);
+      // UI.addBookToList(newBook);
```
Comment (vechicin) [inline index.js]: Please remove this line of commented code to keep a clean codebase.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @belmeetmule ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job implementing all of the required features for this milestone!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -1 +1,66 @@
-# awesome-book-es6
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> awesome-books
```
Comment (Reem-lab) [inline README.md:5]: - [ OPTIONAL ] I suggest adding some information about the project and more description to describe what this app for?

**Review by Reem-lab — APPROVED** [review]
# Hi @yasinabdmahmood, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality ✅
📌 Good commit messages ✅

## Optional suggestions
> * [OPTIONAL] I suggest adding the `MIT.md` file to your project :ok_hand:, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it :wink:
>
> > The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. :100:
>

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**